src/filters/UserFilters.jsx

```diff
@@ -35,6 +35,7 @@ export default class UserFilters extends React.Component {
     }
   }
   updateUserList() {
+    this.props.dispatch(clearUserList());
```

Comment:
This COULD be fired from here, but many other places as well - could it be fired from the actions?